test/features/events/response.test.ts

```diff
@@ -69,6 +69,7 @@ beforeAll(async () => {
 })
 
 afterEach(() => {
+  interceptor.removeAllListeners('response')
```

Comment:
Incredibly flaky test poorly written. Shame on me. 

I'm resetting the response listener because they apparently leak between the tests through the shared interceptor instance. 

I've been scoping request/response listeners within individual tests for some time to prevent this from happening but this is a really old test.